### PR TITLE
Added XML response to web.py

### PR DESCRIPTION
**Pull request by invinciblycool**

### CODE REVIEW

**Comment by invinciblycool** [comment]
Fixes #18.

**Comment by invinciblycool** [comment]
I have converted tabs to spaces but even when I view the changes online, it still shows 8 tabs.
:confused:

**Comment by ashwini0529** [comment]
Hey @invinciblycool See if this helps you? http://paste.ubuntu.com/25679718/

Also, in `from xmljson import badgerfish as bf` , I don't think `xmljson` is a default library.
You might need to update the requirements file.

File: hackr/web.py [hunk 1]
```diff
@@ -1,11 +1,12 @@
 #!/usr/bin/env python
 # -*- coding: utf-8 -*-
 
-import requests
 import json
-from xmljson import badgerfish as bf
 from xml.etree.ElementTree import tostring
 
+import requests
+from xmljson import badgerfish as bf
```
Comment (ashwini0529) [inline hackr/web.py]: `xmljson` needs to be mentioned in the requirements file. Can you please update it?

**Comment by invinciblycool** [comment]
Sorry but I had messed up my remote branch.
Really sorry for the mess.

**Pull request closed without merging**